tutorials/vue/router-push.md

```diff
@@ -0,0 +1,93 @@
+[Vue Router](https://router.vuejs.org/installation.html) has two approaches to navigating a site,
```

Comment:
Let's skip discussing declarative navigation, and instead highlight that `push()` is a way to do programmatic navigation with vue router.